### PR TITLE
Lock Screen widgets (and build 48)

### PR DESCRIPTION
**Pull request by benbacardi**

### CODE REVIEW

File: Intents Extension/IntentHandler.swift [hunk 1]
```diff
@@ -25,6 +25,8 @@ class IntentHandler: INExtension {
             return ListMilestonesIntentHandler()
         case is ConfigurationIntent:
             return ConfigurationIntentHandler()
+        case is CampaignLockScreenConfigurationIntent:
+            return CampaignConfigurationIntentHandler()
```
Comment (alexcfaber) [inline Intents Extension/IntentHandler.swift:29]: But not the Fundraising one?

File: Shared Widgets/LockScreenCircularView.swift [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+struct LockScreenCircularView: View {
+    
+    let campaign: TiltifyWidgetData
+    var shouldShowFullCurrencySymbol: Bool = false
+    var shouldShowGoalPercentage: Bool = false
+    
+    var body: some View {
+        ZStack {
+            if #available(iOSApplicationExtension 16.0, *) {
```
Comment (alexcfaber) [inline Shared Widgets/LockScreenCircularView.swift:19]: No fall back if iOS 16 isn't available?

File: Intents Extension/IntentHandler.swift (hunk 1 above)
Comment (benbacardi) [reply Intents Extension/IntentHandler.swift:29]: That doesn't have anything that needs custom resolution - this is only necessary here to return the list of Campaigns from the database for the "Campaign?" option

File: Shared Widgets/LockScreenCircularView.swift (hunk 2 above)
Comment (benbacardi) [reply Shared Widgets/LockScreenCircularView.swift:19]: This widget is only available on iOS 16 (the `.accessoryXXX` widget types), so the entire view could be wrapped in it really. It'll never get used on any other platform.

File: Shared/Assets.xcassets/Relay 1c 2.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+  "images" : [
+    {
+      "filename" : "Relay 1c 2.png",
```
Comment (alexcfaber) [inline Shared/Assets.xcassets/Relay 1c 2.imageset/Contents.json:4]: Generic comment: I hate spaces in filenames. I know it should work, but I've had so many bugs with them in the past.

**Review by alexcfaber — APPROVED** [review]

File: Shared/Assets.xcassets/Relay 1c 2.imageset/Contents.json (hunk 3 above)
Comment (benbacardi) [reply Shared/Assets.xcassets/Relay 1c 2.imageset/Contents.json:4]: Good shout, I meant to take these out anyway. They're not used, they might as well not be adding to the size of the app.

**Review by alexcfaber — APPROVED** [review]

**Pull request merged**